### ISSUE TITLE
Learning to remember rare events

### ISSUE THREAD

**Issue by singlasahil14** [tensorflow/models#2477]
How am I supposed to run the script to reproduce the results of the paper on 20 way, 1 shot omniglot classification? I tried running it with episode_width=20 and different episode_lengths. But that is giving results nowhere near the paper results.

**Comment by poxvoculi** [comment]
A quick search turned up something from another repository that looks relevant:
https://github.com/AntreasAntoniou/MatchingNetworks

Otherwise, this seems like a StackOverflow question.

**Comment by singlasahil14** [comment]
If the model is not reproducing the results for 20 way classification, is it not a bug?

**Comment by poxvoculi** [comment]
You have not specified exactly which model you are trying to run, nor exactly how.  Unless the model is supposed to be precisely the same as in a publication, invoked the same way with the same data, it seems implausible to argue that a difference in results must be due to an implementation bug.

**Comment by singlasahil14** [comment]
python train.py --memory_size=8192 \
  --batch_size=16 --validation_length=50 \
  --episode_width=20 --episode_length=30
I ran this script. The one-shot accuracy comes out to be around 40%. Is this the expected behavior?

**Comment by lukaszkaiser** [comment]
You need to set --rep_dim=256 to replicate the results (the default is 128 just for speed). Hope that helps!

**Comment by singlasahil14** [comment]
I am afraid it is still not matching the published results.

**Comment by singlasahil14** [comment]
@lukaszkaiser Can you please reopen? The problem is still not solved. It doesn't reproduce the results.

**Comment by lukaszkaiser** [comment]
I reran it some weeks ago and the results were ok, what kind of problems do you see? I'm happy to help if you email me (userid at google dot com), but this github issue tracker is for code bugs -- like "code doesn't run in python 3" -- not ML research problems like "I'm having trouble reproducing on my setup", so let's not reopen it.

**Comment by singlasahil14** [comment]
Can you please send me the exact command line arguments you used for 20-way classification?

**Comment by singlasahil14** [comment]
@lukaszkaiser  Can you please post the exact command line arguments you used here? I will run the code on them and get back to you.

**Comment by lukaszkaiser** [comment]
For me just  --rep_dim=256 works ok. I will not continue the discussion here, please send me an email if you have further trouble reproducing.

**Comment by mmatteo-hub** [comment]
Where can I find the implementation of the Lerning to Remember Rare Events? Since the link on the paper is not linking correctly. Thanks
